src/ontology/oba.Makefile

```diff
@@ -54,7 +54,28 @@ modules/has_part.csv:  $(patsubst %, ../patterns/data/default/%.tsv, $(MODS))
 ## If you need to customize your Makefile, make
 ## changes here rather than in the main Makefile
 
-imports/hp_import.owl: $(TEMPLATEDIR)/external.owl
+#########################################
+### Generating all ROBOT templates ######
+#########################################
+
+TEMPLATESDIR=../templates
+
+TEMPLATES=$(patsubst %.tsv, $(TEMPLATESDIR)/%.owl, $(notdir $(wildcard $(TEMPLATESDIR)/*.tsv)))
+
+$(TEMPLATESDIR)/%.owl: $(TEMPLATESDIR)/%.tsv $(SRC)
+	$(ROBOT) merge -i $(SRC) template --template $< --output $@ && \
+	$(ROBOT) annotate --input $@ --ontology-iri $(ONTBASE)/components/$*.owl -o $@
+
+$(COMPONENTSDIR)/obsoletes.owl: $(TEMPLATESDIR)/replaced.owl
+	$(ROBOT) merge -i $< annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $@
+
+$(COMPONENTSDIR)/synonyms.owl: $(TEMPLATESDIR)/synonyms.owl
+	$(ROBOT) merge -i $< annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $@
+
+$(COMPONENTSDIR)/measured_in.owl: $(TEMPLATESDIR)/measured_in.owl
+	$(ROBOT) merge -i $< annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $@
+
+imports/hp_import.owl: $(TEMPLATESDIR)/external.owl
```

Comment:
Why is all the stuff above in the diff? Maybe you need to update from master?

Reply:
Exactly. In the first PR attempt, I was without permission, during this time a new commit was pushed.